src/schema/v2/verifiedRepresentative/verifiedRepresentative.ts

```diff
@@ -10,13 +10,13 @@ export const VerifiedRepresentativeType = new GraphQLObjectType<any, any>({
     return {
       ...IDFields,
       artist: {
-        type: ArtistType,
+        type: new GraphQLNonNull(ArtistType),
```

Comment:
I believe `ArtistType!` _should_ also work here

Reply:
I don't think that's how it works! I think you do have to use the `GraphQLNonNull` thingy:

<img width="656" alt="Screenshot 2023-08-24 at 10 05 41 AM" src="https://github.com/artsy/metaphysics/assets/79799/a2458518-e9cd-4479-9131-23c6d5278187">

Reply:
Yeah, that seems quite unusual 😓 
![image](https://github.com/artsy/metaphysics/assets/1176374/d80892cc-1160-4bf0-8634-78e9f4f8348d)